packages/datagateway-download/src/downloadStatus/adminDownloadStatusTable.component.tsx

```diff
@@ -0,0 +1,600 @@
+          {!dataLoaded && (
+            <Grid item xs={12}>
+              <LinearProgress color="secondary" />
+            </Grid>
+          )}
+          <Grid item>
+            <Paper
+              style={{
+                height:
+                  'calc(100vh - 64px - 30px - 48px - 48px - (1.75rem + 40px))',
+                minHeight: 230,
+                overflowX: 'auto',
+              }}
+            >
+              <Table
+                columns={[
+                  {
+                    label: t('downloadStatus.username'),
+                    dataKey: 'userName',
+                    filterComponent: textFilter,
+                  },
+                  {
+                    label: t('downloadStatus.preparedId'),
+                    dataKey: 'preparedId',
+                    filterComponent: textFilter,
+                  },
+                  {
+                    label: t('downloadStatus.transport'),
+                    dataKey: 'transport',
+                    filterComponent: textFilter,
+                  },
+                  {
+                    label: t('downloadStatus.status'),
+                    dataKey: 'status',
+                    filterComponent: textFilter,
+                  },
+                  {
+                    label: t('downloadStatus.size'),
+                    dataKey: 'size',
+                    cellContentRenderer: (cellProps) => {
+                      return formatBytes(cellProps.cellData);
+                    },
+                    disableSort: true,
+                  },
+                  {
+                    label: t('downloadStatus.createdAt'),
+                    dataKey: 'createdAt',
+                    cellContentRenderer: (props: TableCellProps) => {
+                      if (props.cellData) {
+                        const date = new Date(props.cellData).toISOString();
+                        return `${date.slice(0, 10)} ${date.slice(11, 19)}`;
+                      }
+                    },
+                    filterComponent: dateFilter,
+                    disableHeaderWrap: true,
+                  },
+                  {
+                    label: t('downloadStatus.deleted'),
+                    dataKey: 'isDeleted',
+                  },
+                ]}
+                sort={sort}
+                onSort={(column: string, order: 'desc' | 'asc' | null) => {
+                  if (order) {
+                    setSort({ ...sort, [column]: order });
+                  } else {
+                    const { [column]: order, ...restOfSort } = sort;
+                    setSort(restOfSort);
+                  }
+                }}
+                data={data}
+                loading={!dataLoaded}
+                loadMoreRows={fetchMoreData}
+                totalRowCount={dataCount}
+                actionsWidth={100}
+                actions={[
+                  function RestoreButton({ rowData }: TableActionProps) {
```

Comment:
I think in the TopCAT admin table there are two buttons, Restore/Delete and Pause/Resume. Whereas we have Delete and Restore/Pause/Resume buttons. Whether it makes sense to have a Pause/Resume button on a deleted download is a different discussion but that option exists in TopCAT. I think we should align more closely with what already exists, or perhaps consult Chris on what configuration of buttons he needs.

Reply:
See my reply to your previous comment.